File: solutions/0268-missing-number/solution.java
class Solution {
    public int missingNumber(int[] nums) {
        List<Integer>ls=new ArrayList<>();
        for(int n:nums)
            ls.add(n);
        for(int i=0;i<=nums.length;i++)
        {
            if(!ls.contains(i))
                return i;
        }
        return 0;
    }
}

File: solutions/0268-missing-number/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n=(nums.length*(nums.length+1))/2,sum=0;
    for(let i=0;i<nums.length;i++)
        sum+=nums[i];
    return n-sum;
};
